README.md

```diff
@@ -2,404 +2,204 @@
-pip install "boxsdk[jwt]"
-```
-
-Instead of instantiating your `Client` with an instance of `OAuth2`,
-instead use an instance of `JWTAuth`.
-
-``` python
-from boxsdk import JWTAuth
-from boxsdk import Client
-
-auth = JWTAuth(
-    client_id='YOUR_CLIENT_ID',
-    client_secret='YOUR_CLIENT_SECRET',
-    enterprise_id='YOUR_ENTERPRISE_ID',
-    jwt_key_id='YOUR_JWT_KEY_ID',
-    rsa_private_key_file_sys_path='CERT.PEM',
-    rsa_private_key_passphrase='PASSPHRASE',
-)
-
-access_token = auth.authenticate_instance()
-client = Client(auth)
-```
-
-This client is able to create application users:
-
-``` python
-ned_stark_user = client.create_user('Ned Stark')
-```
-
-These users can then be authenticated:
-
-``` python
-ned_auth = JWTAuth(
-    client_id='YOUR_CLIENT_ID',
-    client_secret='YOUR_CLIENT_SECRET',
-    user=ned_stark_user,
-    jwt_key_id='YOUR_JWT_KEY_ID',
-    rsa_private_key_file_sys_path='CERT.PEM',
-    rsa_private_key_passphrase='PASSPHRASE'
-)
-ned_auth.authenticate_user()
-ned_client = Client(ned_auth)
+pip install boxsdk~=4.0
 ```
 
-Requests made with `ned_client` (or objects returned from
-`ned_client`'s methods) will be performed on behalf of the newly
-created app user.
-
-## Traditional 3-legged OAuth2
-
-### Get the Authorization URL
-
-``` python
-from boxsdk import OAuth2
-
-oauth = OAuth2(
-    client_id='YOUR_CLIENT_ID',
-    client_secret='YOUR_CLIENT_SECRET',
-    store_tokens=your_store_tokens_callback_method,
-)
-
-auth_url, csrf_token = oauth.get_authorization_url('http://YOUR_REDIRECT_URL')
+To install also extra dependencies required for JWT authentication, use command:
+``` console
+pip install "boxsdk[jwt]~=4.0"
 ```
 
-store_tokens is a callback used to store the access token and refresh
-token. You might want to define something like this:
+Supported Python versions are Python 3.8 and above.
 
-``` python
-def store_tokens(access_token, refresh_token):
-    # store the tokens at secure storage (e.g. Keychain)
-```
+# Getting Started
 
-The SDK will keep the tokens in memory for the duration of the Python
-script run, so you don't always need to pass store_tokens.
+To get started with the SDK, get a Developer Token from the Configuration page of your app in the [Box Developer Console](https://app.box.com/developers/console). 
+Developer Tokens are short-lived and expire after 60 minutes, which is good for testing but not for production use.
+To learn about other authentication methods, see the [Authorization](#authorization) section below.
 
-### Authenticate (Get Access/Refresh Tokens)
+The examples below demonstrate how to authenticate with Developer Token and print names of all items inside a root folder.
 
-If you navigate the user to the auth_url, the user will eventually get
-redirected to <http://YOUR_REDIRECT_URL?code=YOUR_AUTH_CODE>. After
-getting the code, you will be able to use the code to exchange for an
-access token and refresh token.
+## With box_sdk_gen package (recommended)]
```

Comment:
<img width="961" height="161" alt="image" src="https://github.com/user-attachments/assets/2e4bba39-c8d3-419d-b2c7-aad8456d8535" />


Should this header render that way?